sway-fmt-v2/src/items/item_storage.rs

```diff
@@ -150,7 +150,13 @@ fn format_storage(
         let mut value_pairs_iter = fields.value_separator_pairs.iter().peekable();
         for field in value_pairs_iter.clone() {
             // storage_field
-            field.0.format(formatted_code, formatter)?;
+            write!(
+                formatted_code,
+                "{}{} {}",
+                field.0.name.span().as_str(),
+                field.0.colon_token.span().as_str(),
+                field.0.ty.span().as_str(),
```

Comment:
Maybe we should use `ty` formatting here rather than the raw `str` coming from `span` 🤔

Reply:
right you are!